lib/remote_console/rack_server.rb

```diff
@@ -62,7 +62,7 @@ def call(env)
       if WebSocket::Driver.websocket?(env) && same_origin_as_host?(env) && exp.present?
         @logger.info("RemoteConsole connection initiated")
         init_proxy(env, exp[1])
-      elsif same_origin_as_host?(env) && %r{^/ping$}.match(env['REQUEST_URI']).present?
+      elsif %r{^/ping$}.match(env['REQUEST_URI']).present?
```

Comment:
`.match.present?` can just be `.match?` (same above too, technically)

```suggestion
      elsif %r{^/ping$}.match?(env['REQUEST_URI'])
```

I'm also finding it strange that the regex is matching against the string, because it's usually the other way around, so I feel this is even more clear as:

```suggestion
      elsif env['REQUEST_URI'].match?(%r{^/ping$})
```

Reply:
No need to fix for this PR...maybe a follow-up

Reply:
are we guaranteed that `env['REQUEST_URI']` is not nil?

Reply:
I can't see how, but it's also an easy check for .nil? or .blank? if it's needed

Reply:
or we can do `elsif env['REQUEST_URI'].to_s.match?(%r{^/ping$})` to deal with the nil

Reply:
Done